ipatests/test_integration/test_replica_promotion.py

```diff
@@ -417,6 +417,10 @@ def test_manual_renewal_master_transfer(self):
         replica = self.replicas[0]
         replica.run_command(['ipa', 'config-mod',
                              '--ca-renewal-master-server', replica.hostname])
+        # wait for replication to complete before checking on the master
+        ldap = replica.ldap_connect()
+        tasks.wait_for_replication(ldap)
```

Comment:
This looks fine. I did see other calls to the task use a simpler form:
`tasks.wait_for_replication(replica.ldap_connect())`
They are literally equivalent other than styling.

Reply:
> This looks fine. I did see other calls to the task use a simpler form: `tasks.wait_for_replication(replica.ldap_connect())` They are literally equivalent other than styling.

@rcritten thanks, I implemented your suggestion. CI is now green.